src/components/core/ContextSelector/index.tsx

```diff
@@ -8,22 +8,17 @@ import {
     ContextType,
     useCurrentContextTypes,
 } from '@equinor/fusion';
-import {
-    SearchIcon,
-    useDropdownController,
-    Dropdown,
-    Menu,
-    SearchableDropdownSection,
-    SearchableDropdownOption,
-    Spinner,
-    IconButton,
-    CloseIcon,
-    TextInput,
-} from '@equinor/fusion-components';
 import * as styles from './styles.less';
 import classNames from 'classnames';
 
 import contextToDropdownSection, { formattedContextType } from './ContextToDropdownSection';
+import { SearchableDropdownSection, SearchableDropdownOption } from 'components/general/SearchableDropdown';

```

Comment:
Why are these types or patch changes necessary? Is it required to get Jest to work?